import org.junit.Test;
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;
import static org.junit.Assert.assertFalse;

import java.lang.ArithmeticException;


public class TestDemo {
//    @Test  //test registration
//
//    public void userRegisterTest() {
//        assertTrue(UserManagement.getInstance().registerUser(new String[] {"User1", "Password1", "1@qualogy.com", "2000"}));
//        assertTrue(UserManagement.getInstance().registerUser(new String[] {"User2", "Password2", "2@qualogy.com", "0"}));
//        assertFalse(UserManagement.getInstance().registerUser(new String[] {null, "Password3", "3@qualogy.com", "100"}));
//        assertFalse(UserManagement.getInstance().registerUser(new String[] {"User4", null, "4@qualogy.com", "100"}));
//        assertFalse(UserManagement.getInstance().registerUser(new String[] {"User5", "Password5", null, "100"}));
//        assertFalse(UserManagement.getInstance().registerUser(new String[] {"User6", "Password6", "1@qualogy.com", "100"}));
//        assertFalse(UserManagement.getInstance().registerUser(new String[] {"User1", "Password6", "4@qualogy.com", "100"}));
//    }

}